Automattic/wp-calypso
# Stats: Switching Stats module grid to flexbox

Related to #88267

## Proposed Changes

* replacing the old grid with flexbox for the Insights and Subscribers pages
* removing old CSS grid code

## Testing Instructions



* open the live branch
* navigate to the Traffic, Insights and Subscribers pages
* test the page for various screen widths and see if everything aligns well
* Jetpack sites don't have the last module on the Insights page so test both types

## Pre-merge Checklist
